docker/go-control-plane/fallback-control-plane.go

```diff
@@ -0,0 +1,187 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"sync"
+
+	cs "github.com/eugeneo/fallback-control-plane/grpc/interop/grpc_testing/xdsconfig"
```

Comment:
The code generator is different in OSS compared to google3. If at all this code is going to be imported into google3, we need to split imports here into one package for protos and one package for services.

Reply:
Please rename proto imports with a `pb` suffix. See: go/go-style/decisions#import-renaming

Here and elsewhere in this PR.

Reply:
Done